spec/index.bs

```diff
@@ -2733,6 +2739,85 @@ Given a {{GPUCommandEncoder}} |encoder| and the arguments {{GPUTextureCopyView}}
 
 </div>
 
+### <dfn method for=GPUCommandEncoder>copyTextureToTexture(source, destination, copySize)</dfn> ### {#GPUCommandEncoder-copyTextureToTexture}
+
+<div algorithm="GPUCommandEncoder.copyTextureToTexture">
+
+  **Arguments:**
+    - {{GPUTextureCopyView}} |source|
+    - {{GPUTextureCopyView}} |destination|
+    - {{GPUExtent3D}} |copySize|
+
+  **Returns:** void
+
+  Encode a command into the {{GPUCommandEncoder}} that copies data from a sub-region of one
+  or multiple continuous {{GPUTexture}} [=subresources=] to another sub-region of one or
+  multiple continuous {{GPUTexture}} [=subresources=].
+
+  |source| and |copySize| define the region of the source texture [=subresources=].
+
+  |destination| and |copySize| define the region of the destination texture [=subresources=].
+
+</div>
+
+<div algorithm class=validusage>
+
+<dfn abstract-op>copyTextureToTexture Valid Usage</dfn>
+
+Given a {{GPUCommandEncoder}} |encoder| and the arguments {{GPUTextureCopyView}} |source|,
+{{GPUTextureCopyView}} |destination|, {{GPUExtent3D}} |copySize|, let
+
+  - |copy on the whole subresource| be the copy whose parameters |source|, |destination| and |copySize| meets the following conditions:
+    - The width of the [=physical size=] of |source|.{{GPUTextureCopyView/texture}} [=subresource=] at [=mipmap level=] {{GPUTextureCopyView/mipLevel}} must be equal to |copySize|.[=Extent3D/width=].
```

Comment:
can we have a short define for mip(n) of a texture to avoid repeating this 6 times?

Reply:
I've written this part in another way. PTAL, thanks!